pcap.c

```diff
@@ -2067,6 +2068,10 @@ pcap_inject(pcap_t *p, const void *buf, size_t size)
 void
 pcap_close(pcap_t *p)
 {
+	if (p->opt.rfmon_selfstart)
```

Comment:
It has been fixed at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/1c3bf75b09e74efaf63179ef8aba000d2610b415